bin/presubmit.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:async';
+import 'dart:io';
+import 'package:io/io.dart';
+
+Future<Null> main() async {
+  final ProcessManager manager = new ProcessManager();
+  Process spawn;
+
+  print('Formatting config.json...');
+  spawn = await manager.spawn('bin/configlet', ['fmt', '.']);
+  await spawn.exitCode;
+
+  print('Formatting all Dart files...');
+  spawn = await manager.spawn('pub', ['run', 'dart_style:format', '-l', '120', '-w', '.']);
+  await spawn.exitCode;
+
+  print('Running test...');
+  spawn = await manager.spawn('pub', ['run', 'test']);
+  await spawn.exitCode;
+
+  await sharedStdIn.terminate();
```

Comment:
Similar to the previous comment, some lines can be refactored to use a method that spawns and awaits.